Remotion/Web/Development.WebTesting/WebDriver/Factories/Chrome/ChromeBrowserFactory.cs

```diff
@@ -56,6 +56,7 @@ public IBrowserSession CreateBrowser (DriverConfiguration configuration)
       var commandTimeout = configuration.CommandTimeout;
 
       var driver = CreateChromeDriver (out var driverProcessID, commandTimeout);
+      driver.Manage().Timeouts().AsynchronousJavaScript = configuration.AsyncJavaScriptTimeout;

```

Comment:
IMO it's fine to set it here, but this could also be changed in the tests via some extension method for example 😀.

Reply:
You're probably right but I do think doing it like this is cleaner